hash.c

```diff
@@ -2845,9 +2845,9 @@ clear_i(VALUE key, VALUE value, VALUE dummy)
 
 /*
  *  call-seq:
- *    hash.clear -> self
+ *    clear -> self
```

Comment:
The other call-seq seems to standardize on having `hash.` prepended.

```suggestion
 *    hash.clear -> self
```

Reply:
> The other call-seq seems to standardize on having `hash.` prepended.

Doc guide says not:  https://docs.ruby-lang.org/en/master/contributing/documentation_guide_md.html#label-Calling+Sequence+-28for+methods+written+in+C-29.

Will change the others as we go.

Reply:
I see, I fixed all of them in Hash and Array in https://github.com/ruby/ruby/pull/12728 to make it consistent.